README.md

```diff
@@ -0,0 +1,26 @@
+## About
+
+This is a trainer for The Legend of Zelda: Ocarina of Time. It is compatible with version 1.0 and 1.2 of the NTSC release.
+
+## Dependencies
+* The n64-toolchain is required for building it can be found [here](https://github.com/glankk/n64).
+* In installing the n64-toolchain, Code::Blocks IDE should be configured to support targeting the n64. Alonside Code::Blocks, BizHawk should also be installed and configured, including all environment variables.
+
+## Building
```

Comment:
Good thinking. The biggest issue I could see was that the makefile *has* to target 'all' in order to build. I didn't want to make a drastic modification such as re-writing the makefile and including it in gz because I don't know how you want such changes handled, so I'll leave that up to you. I'm also curious as to what changes do you propose will likely be made to the toolchain in the future?